cinema/views.py

```diff
@@ -21,7 +24,7 @@
     MovieSessionDetailSerializer,
     MovieListSerializer,
     OrderSerializer,
-    OrderListSerializer,
+    OrderListSerializer, MovieImageSerializer,
 )
```

Comment:
Code Style: It's good to group imports using `()` if needed. However, it would be better to keep the imports on separate lines for readability. It's also a good practice to sort your imports alphabetically. This makes it easier to quickly scan the list for a particular import.